frontend/src/scenes/insights/filters/BreakdownFilter/BreakdownTag.tsx

```diff
@@ -0,0 +1,84 @@
+import { LemonButton, LemonDivider, LemonInput, LemonTag } from '@posthog/lemon-ui'
+import { useValues } from 'kea'
+import { PropertyKeyInfo } from 'lib/components/PropertyKeyInfo'
+import React, { useState } from 'react'
+import { cohortsModel } from '~/models/cohortsModel'
+import { FilterType } from '~/types'
+import { isAllCohort, isCohort, isPersonEventOrGroup } from './TaxonomicBreakdownFilter'
+
+type BreakdownTagProps = {
+    isHistogramable: boolean
+    setFilters?: (filter: Partial<FilterType>) => void
+    filters: FilterType
+    onClose?: () => void
+    breakdown: string | number
+}
+
+export function BreakdownTag({
+    isHistogramable,
+    setFilters,
+    filters,
+    onClose,
+    breakdown,
+}: BreakdownTagProps): JSX.Element {
+    const { cohortsById } = useValues(cohortsModel)
+
+    const [histogramBinCount, setHistogramBinCount] = useState(filters.breakdown_histogram_bin_count || 10)
```

Comment:
looking at the 10 default, a new edge case comes to mind, say browser versions are [98,99,100], breaking these down into 10 buckets would lead to something like:

[98,98,98,98,99,99,99,100] which is really messed up, because well, the buckets aren't unique 🤯 .

Will fix this on the backend to not have duplicates, but for you & the frontend, this means clarifying how you might not always get 10 buckets, even if you ask for 10 buckets.

Reply:
Quick update: Because we're using the non-exact version of quantiles (non-deterministic and faster, but this doesn't matter much as we only call it once),

this doesn't quite happen, and it breaks down buckets into [98, 98.5], [98.5, 99], [99, 99.5], etc.

The only case where it does still happen is when there's only one breakdown value.

Reply:
I'm not too worried about this issue because a user can always adjust the number of buckets or opt out of histograming if they're hitting this issue.

Reply:
Yeah sounds good. I fixed the backend to not blow up in these cases though, so should be completely fine on the frontend, "we're just doing what you want"